chain/contractWrappers.go

```diff
@@ -122,7 +122,7 @@ func (ph *VotingHandle) NewProcessTxArgs(ctx context.Context, pid [types.Process
 
 	// entity id
 	// for evm censuses the entity id is the snapshoted contract address
-	if processData.EntityId, err = hex.DecodeString(util.TrimHex(processMeta.EntityAddress.String())); err != nil {
+	if processData.EntityId, err = hex.DecodeString(util.TrimHex(processMeta.EntityAddressOwner[0].String())); err != nil {
```

Comment:
What's the type of EntityAddressOwner[0] ? Are we sure it can have the 0x behind the string?

Reply:
Returns 2 hex encoded strings.
Example:
  ```entityAddress_owner: [
    '0x9561C133DD8580860B6b7E504bC5Aa500f0f06a7',
    '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1'
  ],
  ```